static/js/src/entry/circle/CircleWall.vue

```diff
@@ -5,6 +5,16 @@
       <figure class="wall__logo grid_separator--l" aria-hidden="true">
         <img src="./img/circle-logo.png" alt="Circle Membership logo" /></figure
     ></template>
+    <template #subheading="{ group }">
+      <template v-if="group.name === 'Founders\' Circle'"
```

Comment:
Since "Founders' Circle" is also referenced in orderedGroupNames it might be worth considering whether to convert "Founders' Circle" into a constant. Down the line, I can imagine we could easily create a bug if we change the name of "Founders' Circle" and miss the other reference.

Reply:
GREAT point. I ended up doing this: [097e7eb](https://github.com/texastribune/donations/pull/900/commits/097e7eb3886d996ab90e4131f832bbab8eaabe77) to address that. I thought about doing something like [`orderedGroupNames[0]`](https://github.com/texastribune/donations/blob/097e7eb3886d996ab90e4131f832bbab8eaabe77/static/js/src/entry/circle/CircleWall.vue#L35) so the label lived in on place, but we might change the order and forget. It's at least closer to where you'd update and more greppable now, but still redundant. Seems better than what I originally had nevertheless! Thanks for flagging.